sector-builder-ffi/src/api.rs

```diff
@@ -228,16 +229,15 @@ pub unsafe extern "C" fn sector_builder_ffi_get_staged_sectors(
 }
 
 /// Generates a proof-of-spacetime for the given replica commitments.
-///
 #[no_mangle]
-pub unsafe extern "C" fn sector_builder_ffi_generate_post(
+pub unsafe extern "C" fn sector_builder_ffi_generate_candidates(
     ptr: *mut SectorBuilder,
     flattened_comm_rs_ptr: *const u8,
     flattened_comm_rs_len: libc::size_t,
     challenge_seed: &[u8; 32],
     faults_ptr: *const u64,
     faults_len: libc::size_t,
-) -> *mut types::GeneratePoStResponse {
+) -> *mut types::GenerateCandidatesResponse {
     catch_panic_response(|| {
         init_log();
 
```

Comment:
should probably be `info!("generate_candidates: {}", "start");`

Reply:
fixed